charts/prometheus-stackdriver-exporter/templates/deployment.yaml

```diff
@@ -73,10 +73,10 @@ spec:
               mountPath: /etc/secrets/service-account/
           {{- end}}
           args:
-            - --google.project-id={{ .Values.stackdriver.projectId }}
+            - --google.project-ids={{ .Values.stackdriver.projectId }}
             - --monitoring.metrics-interval={{ .Values.stackdriver.metrics.interval }}
             - --monitoring.metrics-offset={{ .Values.stackdriver.metrics.offset }}
-            - --monitoring.metrics-type-prefixes={{ .Values.stackdriver.metrics.typePrefixes | replace " " "" }}
```

Comment:
also here

Reply:
old flag is still supported, new flag is **repeatable**